src/schematools/types.py

```diff
@@ -754,8 +869,17 @@ def is_nested_table(self) -> bool:
     def is_through_table(self) -> bool:
         """
         Checks if field is a possible through table.
+
+        XXX: What if source is not temporal, but target is temporal?
+        Do we have a through table in that case?
         """
-        return self.is_array and self.nm_relation is not None
+
+        return (
+            self.is_array
+            and self.nm_relation is not None
+            or self._parent_table.is_temporal
+            and self.relation is not None
```

Comment:
graag haakjes, want is het `(a AND b) OR (`c AND c`) ?

Reply:
Maar, `and` heeft hogere precedence dat `or` toch?

Reply:
`and` does indeed have a higher precedence then `or` though I would use brackets as well

Reply:
Ok, I'll add brackets for readability.